Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayer.h

```diff
@@ -253,6 +255,8 @@ class WEBCORE_EXPORT CoordinatedGraphicsLayer : public GraphicsLayer {
 
     RefPtr<AnimatedBackingStoreHost> m_animatedBackingStoreHost;
     RefPtr<CoordinatedGraphicsLayer> m_backdropLayer;
+
+    std::optional<double> estimatedBackingStoreMemory;
```

Comment:
I'm not sure why you use an optional here. A double initialized to 0 should be enough, as 0 is what you're returning when the optional is not initialized. I mean, you can use the optional, but it's an overkill in this case.

Reply:
You are right, I will use just double for that.